src/components/search-form/search-form.js

```diff
@@ -4,6 +4,7 @@ import Link from 'react-router-dom/Link';
 import capitalize from 'lodash/capitalize';
 import isEqual from 'lodash/isEqual';
 import SearchField from '@folio/stripes-components/lib/structures/SearchField';
+import Button from '@folio/stripes-components/lib/Button';
```

Comment:
Why not 

```js
import { Button } from '@folio/stripes-components';
```

Reply:
We can do that with a Button, but we're not importing anything else from `stripes-components`  directly off the package.

The index isn't kept up to date: https://github.com/folio-org/stripes-components/blob/master/index.js